struts2-jquery-plugin/src/main/resources/template/jquery/autocompleter.ftl

```diff
@@ -18,27 +18,40 @@
  * under the License.
  */
 -->
+<#assign hasFieldErrors = parameters.widgetname?? && fieldErrors?? && fieldErrors[parameters.widgetname]??/>
+<input type="hidden"
+    <#if parameters.widgetid?if_exists != "">
+        id="${parameters.widgetid?html}"<#rt/>
+    </#if>
+    <#if parameters.nameValue??>
+        value="<@s.property value="parameters.nameValue"/>"<#rt/>
+    </#if>
+    <#if parameters.widgetname?if_exists != "">
+        name="${parameters.widgetname?html}"<#rt/>
+    </#if>
+    <#if parameters.cssClass?has_content && !(hasFieldErrors && parameters.cssErrorClass??)>
+        class="${parameters.cssClass?html}"<#rt/>
+    <#elseif parameters.cssClass?has_content && (hasFieldErrors && parameters.cssErrorClass??)>
+        class="${parameters.cssClass?html} ${parameters.cssErrorClass?html}"<#rt/>
+    <#elseif !(parameters.cssClass?has_content) && (hasFieldErrors && parameters.cssErrorClass??)>
+        class="${parameters.cssErrorClass?html}"<#rt/>
+    </#if>
+    <#if parameters.cssStyle?has_content && !(hasFieldErrors && (parameters.cssErrorStyle?? || parameters.cssErrorClass??))>
+        style="${parameters.cssStyle?html}"<#rt/>
+    <#elseif hasFieldErrors && parameters.cssErrorStyle??>
+        style="${parameters.cssErrorStyle?html}"<#rt/>
+    </#if>
+    <#if parameters.disabled?default(false)>
+        disabled="disabled"<#rt/>
+    </#if>
+/>

```

Comment:
> EDIT : for the control header, it's not really a pb because it's an hidden field, that only contains the value to be submitted. the control header will still be above the visible textfield/combobox.

You're right, it shouldn't matter since it's a hidden field.

I'll leave it up to jogep to decide though.